src/screens/Login/handler.js

```diff
@@ -0,0 +1,49 @@
+import { useContext, useState } from 'react';
+
+import { AuthContext } from 'contexts/auth';
+import Api from 'utils/Api';
+
+const Handler = () => {
+  const [requestSuccess, setRequestSuccess] = useState(false);
+  const [error, setError] = useState(null);
+  const { dispatch } = useContext(AuthContext);
+
+  const handleSubmit = async (values) => {
+    try {
+      await Api.post('api/v1/oauth/token', {
+        grant_type: 'password',
+        email: values.email,
+        password: values.password,
+        client_id: process.env.REACT_APP_CLIENT_ID,
+        client_secret: process.env.REACT_APP_CLIENT_SECRET,
+      })
+        .then(function (response) {
+          if (response.status === 200) {
+            setRequestSuccess(true);
+
+            return response.data;
+          }
+        })
+        .then((resData) => {
+          dispatch({
+            type: 'AUTH',
+            payload: resData.data,
+          });
+        });
+    } catch (error) {
+      if (error.response.status === 400) {
```

Comment:
@rafayet-monon Should it return with `401`? 🤔

Reply:
The api returns 400 in this case -

![image](https://user-images.githubusercontent.com/14927672/95938822-7d4c3c80-0dfc-11eb-823a-fa026b0d3298.png)